include/tester/ssp/SoftphoneManager.hpp

```diff
@@ -78,19 +82,17 @@ class SoftphoneManager
         for(int i = 0; i < amount; i++)
         {
             args.id = std::to_string(i + START_URI);
-            Softphone *sp = new Softphone(args);
-            _softphones.insert({args.id, sp});
-            pj_thread_sleep(MILLISECONDS_TO_SECONDS);
+            _softphones.emplace_back(std::make_shared<Softphone>(args));
+            pj_thread_sleep(MILLISECONDS_TO_SECONDS/TEST_SLEEP_DURATION);
```

Comment:
Just remember that the `TEST_SLEEP_DURATION` is currently set to `100` and it will be quite long, might wanna shorten it to like 10 seconds or something

Reply:
done